api_app/analyzers_manager/observable_analyzers/onionscan.py

```diff
@@ -0,0 +1,57 @@
+"""
+This IntelOwl module adds onionscan utility support to scan tor .onion domains.
+"""
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import json
+import subprocess
+from shutil import which
+
+from api_app.analyzers_manager import classes
+from api_app.exceptions import AnalyzerRunException
+
+
+class OnionScan(classes.ObservableAnalyzer):
+    """
+    Scans domains with onionscan for misconfigurations and leaks
```

Comment:
Either remove this doc string or make it the same as written in the `description` attribute in the JSON configuration.

Reply:
Ok, Sir.